lib/gen_stage.ex

```diff
@@ -1726,7 +1770,7 @@ defmodule GenStage do
     end
   end
 
-  defp init_producer(mod, opts, state) do
+  defp init_producer(mod, opts, state, continue \\ :no_continue) do
```

Comment:
I don't like to use default arguments to private functions. We can directly pass `:no_continue` when calling this function. :)